source/code/plugins/oms_diag_lib.rb

```diff
@@ -126,6 +198,15 @@ def GetCurrentFormattedTimeForDiagLogs()
           Time.now.utc.strftime("%Y-%m-%dT%H:%M:%S.%6NZ")
       end
 
+      # Method used to get major minor version parts as array of omsagent version
+      def GetVersionAsArray(versionStr)
```

Comment:
I think this should be a little more explicit as major.minor.patch-build; see our [bundle code](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/bundle/bundle_skel.sh#L133) for how we compare version numbers there.